src/view/document.js

```diff
@@ -109,14 +109,47 @@ export default class Document {
 	 * not be fixed in the new document tree state.
 	 *
 	 * View post-fixers are useful when you wants to update view structure whenever it changes, for instance add some classes
-	 * to elements based on the view structure or selection. However, is you need DOM elements to be already updated, use
+	 * to elements based on the view structure or selection. However, if you need DOM elements to be already updated, use
 	 * {@link module:engine/view/view~View#event:render render event}.
 	 *
 	 * As a parameter, a post-fixer callback receives a {@link module:engine/view/downcastwriter~DowncastWriter downcast writer}
 	 * instance connected with the executed changes block.
 	 *
-	 * Note that registering a post-fixer won't re-render the editor's view. If the view should change after registering the post-fixer then
-	 * it should be done manually calling `view.forceRender();`.
+	 * Here is an example which wraps all spaces into spans to add a gray background to all of them:
```

Comment:
Is this correct? I remember you said recently that view post-fixers should not change the structure? Or does it apply only to containers?

Reply:
Komentujesz jakąś starą wersję. Tego już tam nie ma.

Reply:
LOL :D I kept this tab open for just too long :P